src/feetech_ros2_driver.cpp

```diff
@@ -131,18 +133,32 @@ hardware_interface::return_type FeetechHardwareInterface::read(const rclcpp::Tim
 
 hardware_interface::return_type FeetechHardwareInterface::write(const rclcpp::Time& /* time */,
                                                                 const rclcpp::Duration& /* period */) {
-  const auto positions = ranges::views::zip(hw_positions_, joint_offsets_) |
-                         ranges::views::transform([&](const auto tuple) {
-                           auto [position, offset] = tuple;
-                           return feetech_driver::from_radians(position) + offset;
-                         }) |
-                         ranges::to_vector;
-  const auto write_result = communication_protocol_->sync_write_position(
-      joint_ids_, positions, std::vector(joint_ids_.size(), 2400), std::vector(joint_ids_.size(), 50));
-  if (!write_result) {
-    spdlog::error("FeetechHardwareInterface::write -> {}", write_result.error());
-    return hardware_interface::return_type::ERROR;
+  // Create vectors only for joints that have command interfaces
+  std::vector<uint8_t> commanded_joint_ids;
```

Comment:
nit: Does it make sense to make this a member variable and set it in the `export_command_interfaces` function?